examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf

```diff
@@ -0,0 +1,18 @@
+resource "awscc_efs_access_point" "this" {
+  file_system_id = "fs-0dbbfe225fc860016" #awscc_efs_file_system.this.id
```

Comment:
remove comment and add `awscc_efs_file_system` resource block

Reply:
Thanks @rahul-amzn , Added awscc_efs_file_system resource and dynamic reference.